vector/src/main/java/im/vector/util/RageShake.java

```diff
@@ -74,10 +74,11 @@ public RageShake(Context context) {
         }
     }
 
+    // For the Tchap app, the rage shake feature is disabled in the preferences manager.
```

Comment:
Please comment the `promptForReport` call (line 189) instead of its definition.
The current code could not compile